test/nodetool/test_cluster.py

```diff
@@ -0,0 +1,375 @@
+#
```

Comment:
Let's call this file `test_cluster_repair.py`.
With pytest, it is preferred to have smaller files with reduced scope (testing a single command), so future `cluster` subcommands should get their own file too.